### ISSUE TITLE
Beliefs' bugs

### ISSUE THREAD

**Issue by Riglav** [carriontrooper/Alpha-Frontier#83]
It's strange that nobody noticed it earlier, but it's look like that effects from "Religious Documents" (+34% Natural religion spread in cities following our religion; +34% Natural religion spread in cities following our religion after discovering "Info Networks") working only for cities that not belong to it's owner: I have basic +9 pressure for city on my own cities which are still not following my religion, but I have large pressure bonuses for City-States and other nations. Meanwhile, looks like the "Religious Unity" (+100% Natural religion spread in City-State cities) belief working on ALL cities, not City-States only, because enemy with that belief has +18 pressure from only one city, when I (with "Religious Documents") have the same value from two. And also, how do bonuses from "Work Ethos" and, especially, from "Specialist Reverence" working? Do they give fixed production bonuses for every city or they give bonuses for every 3 follovers ("Work Ethos") and for every Specialist ("Specialist Reverence")?

**Comment by carriontrooper** [comment]
Hmm, weird stuff happening, lemme check

**Comment by carriontrooper** [comment]
Updated, changed em a bit.

Also, for work ethos, its effect only happens if your city has over 3 followers, while specialist reverence's is when you have at least 1 specialists. Flat bonus. Then again, it wasn't supposed to - now it grants 1 prod per spec.

**Comment by Riglav** [comment]
Yeah it's better now, but looks like "Faith in Science" doesn't work at all. Or, maybe, it should add:+1 faith for every 5_0_ science? Because I have the city with 26 science, but there's no change in faith income.

**Comment by carriontrooper** [comment]
Changed Faith in Science into +2 Faith per Scientist in cities with your religion, tell me if it works out
